doc/ext/doctest.rst

```diff
@@ -176,6 +173,16 @@ The following is an example for the usage of the directives.  The test via
 
 There are also these config values for customizing the doctest extension:
 
+.. confval:: doctest_default_flags
+
+   By default, these options are enabled: ``ELLIPSIS`` (allowing you
```

Comment:
So would I — or even better,  a set. Still, in doctest it isn't, and I'd rather keep it consistent with that.